### PR TITLE
FIX for #20: Cannot find icon: /Resources/icons/Maker_workbench_icon.svg

### PR DESCRIPTION
**Pull request by allanlaal**

### CODE REVIEW

**Comment by allanlaal** [comment]
tested and works!

**Comment by davidmubernal** [comment]
Thank you very much @allanlaal  for your contribution.
I haven't had time to check it out yet. I hope to make time to test and merge it this weekend.

**Review by tjleon1 — CHANGES_REQUESTED** [review]
This did not actually work for me when I tested it.  I found that properly enunciating the filepath worked:

`    Icon = FreeCAD.getUserAppDataDir()+"Mod"+"/MakerWorkbench/Resources/icons/Maker_workbench_icon.svg"
`

**Comment by evanhorn** [comment]
This also did not work for me, so I (shamelessly) borrowed from other workbenches to find a solution.

File: InitGui.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 class MakerWorkbench (Workbench):
     """Maker Wokbench to create objects"""
     #  Icon in XPM 16x16
-    Icon = "/Resources/icons/Maker_workbench_icon.svg"
+    Icon = "./Resources/icons/Maker_workbench_icon.svg"
```
Comment (tjleon1) [inline InitGui.py:30]: This is the way to properly enunciate the filepath:
`Icon = FreeCAD.getUserAppDataDir()+"Mod"+"/MakerWorkbench/Resources/icons/Maker_workbench_icon.svg"`

**Pull request closed without merging**